src/library/Uno.Material/Styles/Controls/v2/RadioButton.xaml

```diff
@@ -1,203 +1,257 @@
-										  ContentTemplate="{TemplateBinding ContentTemplate}"
-										  ContentTransitions="{TemplateBinding ContentTransitions}"
-										  Content="{TemplateBinding Content}"
-										  Margin="{TemplateBinding Padding}"
-										  HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"
-										  VerticalAlignment="{TemplateBinding VerticalContentAlignment}"
-										  AutomationProperties.AccessibilityView="Raw"
-										  TextWrapping="Wrap"
-										  Grid.Column="1" />
-					</Grid>
-				</ControlTemplate>
-			</Setter.Value>
-		</Setter>
-	</Style>
-
-	<Style x:Key="MaterialSecondaryRadioButtonStyle"
-		   TargetType="RadioButton"
-		   BasedOn="{StaticResource MaterialRadioButtonStyle}">
-
-		<Setter Property="Background"
-				Value="{ThemeResource SecondaryBrush}" />
-	</Style>
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+                    xmlns:android="http://uno.ui/android"
+                    xmlns:ios="http://uno.ui/ios"
+                    xmlns:wasm="http://uno.ui/wasm"
+                    mc:Ignorable="ios android wasm">
+
+    <ResourceDictionary.ThemeDictionaries>
+		<ResourceDictionary x:Key="Default">
+			<!--#region Foreground-->
+            <StaticResource x:Key="RadioButtonForeground" ResourceKey="OnBackgroundMediumBrush" />
+            <StaticResource x:Key="RadioButtonForegroundPointerOver" ResourceKey="OnBackgroundMediumBrush" />
+            <StaticResource x:Key="RadioButtonForegroundPressed" ResourceKey="OnBackgroundMediumBrush" />
+            <StaticResource x:Key="RadioButtonForegroundDisabled" ResourceKey="OnBackgroundDisabledBrush" />
+            <!--#endregion-->
+			
+            <!--#region Stroke-->
+            <StaticResource x:Key="RadioButtonOuterEllipseStroke" ResourceKey="OnBackgroundMediumBrush" />
+            <StaticResource x:Key="RadioButtonOuterEllipseStrokeDisabled" ResourceKey="OnBackgroundDisabledBrush" />
+            <StaticResource x:Key="RadioButtonOuterEllipseCheckedStroke" ResourceKey="PrimaryBrush" />
+            <StaticResource x:Key="RadioButtonOuterEllipseCheckedStrokeDisabled" ResourceKey="OnBackgroundDisabledBrush" />
+            <!--#endregion-->
+
+            <!--#region Fill-->
+            <StaticResource x:Key="RadioButtonOuterEllipseFill" ResourceKey="SystemControlTransparentBrush" />
+            <StaticResource x:Key="RadioButtonOuterEllipseFillDisabled" ResourceKey="SystemControlTransparentBrush" />
+            <StaticResource x:Key="RadioButtonOuterEllipseCheckedFill" ResourceKey="PrimaryBrush" />
+            <StaticResource x:Key="RadioButtonOuterEllipseCheckedFillDisabled" ResourceKey="OnBackgroundDisabledBrush" />
+            <!--#endregion-->
+
+            <!--#region HoverRing-->
+            <StaticResource x:Key="MaterialRadioButtonHoverRingFillPointerOver" ResourceKey="PrimaryHoverBrush" />
```

Comment:
The HoverRing Ellipse isn't something that's used in Fluent to indicate a PointerOver or Pressed/Focused state. That's why I added the "Material" prefix